pkg/asset/asset.go

```diff
@@ -37,6 +37,9 @@ const (
 	AssetPathManifests                   = "manifests"
 	AssetPathKubelet                     = "manifests/kubelet.yaml"
 	AssetPathProxy                       = "manifests/kube-proxy.yaml"
+	AssetPathProxySA                     = "manifests/kube-proxy-sa.yaml"
+	AssetPathProxyRoleBinding            = "manifests/kube-proxy-role-binding.yaml"
+	AssetPathServiceAccountKubeConfig    = "manifests/service-account-kubeconfig.yaml"
```

Comment:
nit: everywhere else, we say the name followed by the type. `KubeconfigServiceAccount` and `kubeconfig-sa.yaml`?

Reply:
See below, got confused. Its not a service account.